### PR TITLE
Add Social Distancing dataset from hdx

### PR DESCRIPTION
**Pull request by cryptox31**
Feat: https://trello.com/c/hAdpQrOP/90-social-distancing-policies-subtask-1

**Description of the dataset:** 
>The #COVID19 Government Measures Dataset puts together all the measures implemented by governments worldwide in response to the Coronavirus pandemic. Data collection includes secondary data review. The researched information available falls into five categories: Social distancing Movement restrictions Public health measures Social and economic measures Lockdowns Each category is broken down into several types of measures
- Added hdx api connector and formatter
- hdx connector connects to hdx, and pull acaps COVID19 Government Measures Dataset as a excel
- hdx formatter removes unnecessary columns and returns a dataframe

### CODE REVIEW

File: task_geo/data_sources/hdx/__init__.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from task_geo.data_sources.hdx.hdx_connector import acap_connector
+from task_geo.data_sources.hdx.hdx_formatter import acap_formatter
+
+
+def hdx_api():
```
[the remark this answers is not in the document]
Comment (cryptox31) [reply task_geo/data_sources/hdx/__init__.py:5]: Datasource - hdx
We could be using 2 more datasets from the same data source
I have used the org instead. Does that make sense or should I change that to the datasource itself?

[the remark this answers is not in the document]
Comment (ManuelAlvarezC) [reply task_geo/data_sources/hdx/__init__.py:5]: Let's see:

You are telling me that:
1. hdx is the name of the organization.
2. acap is the provider of the data.
And you maybe planning to use more datasets from hdx, which may not be provided by acap, I'm I right?

So, the naming should be, for this data_source:
- hdx_acap: for the datasource.
-  hdx_acap_connector: for the connector.
- hdx_acap_formatter: for the formatter.

In the future, if you plan to use a dataset provided by the organization xxx, but available from hdx, then use:
 - hdx_xxx: for the datasource.
-  hdx_xxx_connector: for the connector.
- hdx_xxx_formatter: for the formatter.

Does this make sense to you?

[the remark this answers is not in the document]
Comment (cryptox31) [reply task_geo/data_sources/hdx/__init__.py:5]: Yes, this makes sense. I am updating my code

**Comment by ManuelAlvarezC** [comment]
@cryptox31, I see you have pushing commits, but the naming issues I mentioned are not resolved. or is github trolling me?

**Comment by cryptox31** [comment]
@ManuelAlvarezC Some updates have been added yesterday

File: task_geo/data_sources/__init__.py [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+from task_geo.data_sources.hdx import hdx_acap_api
 from task_geo.data_sources.noaa import noaa_api
 
 AVAILABLE_DATA_SOURCES = {
     'noaa_api': noaa_api,
     'cds': cds,
     'us_census': us_census,
-    'nyt': nyt
+    'nyt': nyt,
+    'hdx': hdx_acap_api
```
Comment (ManuelAlvarezC) [inline task_geo/data_sources/__init__.py:12]: The name of the data source should be `hdx_acap`, not `hdx_acap`, the display name should be also the same as the datasource name, this line should be:

```python
AVAILABLE_DATA_SOURCES = {
    ...
    'hdx_acap': hdx_acap,
}
```
This means the module name, and the data source name should be changed too.

**Pull request merged**